### ISSUE TITLE
buttons / icons

### ISSUE THREAD

**Issue by Goeny** [Real-Gecko/Filemin#1]
Only icons for controls seem a bit unclear sometimes.
Perhaps place the action text directly underneath or next to the icon for faster usability?

I have made an example with bootstrap buttons + font-awesome icons to try making clear what i mean. This offcourse is not how it should be done, just an example.

![filemin-buttons](https://cloud.githubusercontent.com/assets/3872395/7498655/d59b6946-f422-11e4-8629-4aa9bcf3961a.png)

**Comment by Real-Gecko** [comment]
Yep, I think you're right. Your choice for icons is pretty cool. I think I will implement 'em right as you posted :)

**Comment by iliaross** [comment]
+1

**Comment by Real-Gecko** [comment]
http://www.bootply.com/HWWgEY1t9c
What if it'll look like this?

**Comment by Goeny** [comment]
I would change the "X" for deleting files to the trashcan. But that is a personal preference.
Also i would add the text to the button for clarity. A bit shorter then the tooltip, with the tooltip giving a bit extra info.
i.e: 

``` html
<a href="#" 
class="btn btn-xs btn-primary" 
title="Delete all selected files and folders" 
data-toggle="tooltip" 
data-placement="bottom">
    <i class="fa fa-trashcan"></i> Delete
</a>
```

**Comment by Real-Gecko** [comment]
http://www.bootply.com/V0vZ00RXOG
As you can see adding text to button makes toolbar not as compact as it was before even if text is under icon.
I think extending description is better option while keeping only icons on buttons/links.

**Comment by iliaross** [comment]
+1

**Comment by Goeny** [comment]
You are right. Smaller toolbar looks better.

**Comment by iliaross** [comment]
I vote for blue white icons that were on top! I think it's not that big as `table_icons` and `generate_icon` (that was my bad to think to put it there, sorry!)

Filemin is awesome!

**Comment by Real-Gecko** [comment]
FIxed in version 0.9
